scripts/python/repos.py

```diff
@@ -570,7 +570,7 @@ def sync(self, pkg_list, alt_url=None):
             host = re.search(r'http://([^/]+)', alt_url).group(1)
             for pkg in pkg_list2:
                 print(pkg)
-                cmd = (f'python2.7 -m pip download --platform ppc64le --no-deps '
+                cmd = (f'python -m pip download --platform ppc64le --no-deps '
```

Comment:
We need to add '--python-version 27' to get the python 2.7 versions instead of python3 versions

Reply:
I just pushed a commit with the '--python-version 27' argument added (haven't tested it, though).